package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "theme-liquid-docs",
+  "version": "1.0.0",
```

Comment:
What do you think about including the `"private": true` property to clarify this repository is not intended to be published as a NPM package?